src/modules/site/base/views/user.py

```diff
@@ -41,6 +45,39 @@ def user_register():
   return render_template('user/register.html', **locals())
 
 
+@user_bp.route("/recover", methods=["GET", "POST"])
+def user_recover():
+  """ Enter an email address for a user to send a password reset link """
+  title = "Recover User"
+  disable_parent_breadcrumb = True
+  form = RecoverUserForm(request.form)
+
+  if request.method == 'POST' and form.validate():
+    email = request.form.get("email")
+    users = get_user_by_email(email)
+
+    if users and users[0]:
+      user = users[0]
+      token = create_one_time_token(id=user.get('username'), destination=url_for('user.user_reset_password'))
+      password_reset_link = url_for('auth.token', token=token, _external=True)
+      try:
+        send_email({
+          "from": "no-reply@elegansvariation.org",
+          "to": [ email ],
+          "subject": "CeNDR Password Reset",
+          "text": PASSWORD_RESET_EMAIL_TEMPLATE.format(email=email, password_reset_link=password_reset_link)
+        })
+        logger.info(f"Sent password reset email: {email}, link: {password_reset_link}")
+      except Exception as err:
+        logger.error(f"Failed to send email: {err}")

```

Comment:
let's discuss this -- it appears that you are trying to avoid leaking any details if an email exists in the system, which is an excellent best practice. 
In this particular case, the email subsystem may actually throw an error (e.g. invalid API, etc), and that should be communicated to the user with something along the lines of "Unable to reset your credentials at this time. Please try again later." as a flash message.